### PR TITLE
307.requirements

### PR DESCRIPTION
**Pull request by meejah**
See #307

(Note that this based on some other PRs that fix CI and other problems)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,18 +36,22 @@ Usage
 =====
 
 Magic-Folder is configured via the ``magic-folder`` command-line tool.
-This tool manipulates the ``magic_folders.yaml`` file in a Tahoe-LAFS' node's private area.
-See the configuration documentation for full details.
 
-Once Magic-Folder is configured,
-functionality is provided by running a long-lived magic-folder side-car for the Tahoe-LAFS node.
-Start the side-car process using the ``magic-folder`` command line too::
+Magic-Folder configuration is kept in a directory. Inside this directory is a database for global configuration and sub-directories to track state and temporary space for each actual magic-folder including a configuration database. All databases are SQLite.
```
Comment (exarkun) [inline README.rst:40]: Let's use semantic linefeeds for magic-folder docs.

File: README.rst [hunk 2]
```diff
@@ -36,18 +36,22 @@ Usage
...
 
-  magic-folder --node-directory /path/to/tahoe-lafs/node run
+A running Magic-Folder needs to have access to a Tahoe-LAFS client that it may use to perform operations in the Tahoe-LAFS Grid. This is referenced by the "node directory" of the Tahoe-LAFS client although actual operations are via the Tahoe-LAFS WebUI.
 
-As long as this side-car is running,
-whatever magic folders are configured will be functional.
+There are two ways to create a new Magic Folder instance (that is, the configuration required). Create a fresh one with ``magic-folder create`` or migrate from a Tahoe-LAFS 1.14.0 or earlier instance with ``magic-folder migrate``.
+
+Once a Magic-Folder is configured functionality is provided by running a long-lived magic-folder daemon. This process is run using the ``magic-folder`` command line too::
```
Comment (exarkun) [inline README.rst:46]: typo in `tool` at the end

File: gridsync-requirements.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+
```
Comment (exarkun) [inline gridsync-requirements.md:1]: Let's put this list on the upstream (I guess?) issue instead.

**Review by exarkun — CHANGES_REQUESTED** [review]
Thanks.  Some of the changes here look like improvements but they're not really related to the referenced issue.  For the rest, I'd like the info to appear elsewhere.  So how about re-targeting this PR at a new issue for the README improvements and move the rest of the changes?

File: README.rst (hunk 1 above)
Comment (meejah) [reply README.rst:40]: I kind of thought I was .. I guess I got the granularity wrong? Are you asking for a linefeed per "."?

**Comment by meejah** [comment]
okay, yeah .. the README changes i think grew here because at least it (now) accurately describes what exists now (and hopefully that's good for gridsync) .. will move the actual-req's to gitlab.

**Comment by meejah** [comment]
okay, moved req's to gitlab issue and filed https://github.com/LeastAuthority/magic-folder/pull/316 instead for README stuff

File: README.rst (hunk 1 above)
Comment (exarkun) [reply README.rst:40]: Yea, I think that's the minimum number of linefeeds I interpret "semantic linefeeds" as calling for.  Sometimes I also put them in after `,` or other intra-sentence punctuation.  It starts to get a bit more subjective when you go beyond `.` (so I wouldn't generally ask for *more* newlines than that in a review).

**Pull request closed without merging**